squander/src-cpp/gates/Gate.cpp

```diff
@@ -696,12 +696,10 @@ Gate::apply_kernel_from_right( Matrix& u3_1qbit, Matrix& input ) {
 }
 
 #ifdef _WIN32
-void sincos(double x, double *s, double *c)
+Gate::void sincos(double x, double *s, double *c)
```

Comment:
`Gate:` should be removed from this Windows version or it will break, it is a global function, not a class function.